guidance/be-nice.md

```diff
@@ -0,0 +1,53 @@
+# Contributors Code of Conduct
+
+The .NET Foundation was created to foster an open, innovative and inclusive
+community around open source .NET. To clarify expected behaviour in our
+communities we have adopted the [Contributor Covenant](http://www.dotnetfoundation.org/code-of-conduct).
+This code of conduct has been adopted by [many other open source communities](http://contributor-covenant.org/adopters/)
+and we feel it expresses our values well. 
+
+For a full copy of the [code of conduct](http://www.dotnetfoundation.org/code-of-conduct),
+see the .NET Foundation Website http://www.dotnetfoundation.org/code-of-conduct
+
+## Applying the Code of Conduct to your project
+
+First of all, it is important that you read the [code of conduct](http://www.dotnetfoundation.org/code-of-conduct)
```

Comment:
The "when to be on the look out" part reads a bit odd. I think the guidance is more like:

- Go about your business as an awesome OSS developer.
- If you experience or see something that feels wrong or that makes you want to step away from the project, do this.